homeassistant/components/climate/ecobee.py

```diff
@@ -269,6 +271,7 @@ def device_state_attributes(self):
             "fan": self.fan,
             "mode": self.mode,
             "operation": operation,
+            ATTR_AUX_HEAT: aux_heat,
```

Comment:
I don't get why this is used, there is a property that needs to be used instead: `is_aux_heat_on`. `device_state_attributes` should only contain attributes that are not already exposed by the base entity class: https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/climate/__init__.py#L411-L466